test/integration/garbage_collector_test.go

```diff
@@ -139,6 +143,9 @@ func TestCascadingDeletion(t *testing.T) {
 	t.Log("This test is failing too much-- lavalamp removed it to stop the submit queue bleeding")
 	return
 	gc, clientSet := setup(t)
+	oldEnableGarbageCollector := registry.EnableGarbageCollector
+	registry.EnableGarbageCollector = true
```

Comment:
ok, I made it through everything except for this test file. hopefully that's enough to keep you busy.